docs/user-guide/hypershift-agent-service-config.md

```diff
@@ -0,0 +1,141 @@
+2. Hive [installation](https://github.com/javipolo/openshift-assisted-installer-tests/tree/4515edfc9d60a587cabd3bcfdd77826cab8cc02d/ai-in-zero-worker-hypershift#install-hypershift) 
+3. Create [hypershift cluster](https://github.com/javipolo/openshift-assisted-installer-tests/tree/4515edfc9d60a587cabd3bcfdd77826cab8cc02d/ai-in-zero-worker-hypershift#create-hypershift-cluster)
+
+## Deploying the hub cluster
+### Set up our environment
+```bash
+AGENT_NAMESPACE=myclusters
+CLUSTERNAME=acm-1
+CLUSTER_NAMESPACE=${AGENT_NAMESPACE}-${CLUSTERNAME}
+BASEDOMAIN=redhat.com
+SSHKEY=~/.ssh/id_rsa.pub
+```
+
+### Deploy Hypershift
+```bash
+export HYPERSHIFT_IMAGE=quay.io/hypershift/hypershift-operator:4.11
+
+alias hypershift="podman run --net host --rm --entrypoint /usr/bin/hypershift -e KUBECONFIG=/credentials/kubeconfig -v /root/.ssh:/root/.ssh -v /tmp/kubeconfig:/credentials/kubeconfig -v /tmp/pull-secret.json:/tmp/pull-secret.json $HYPERSHIFT_IMAGE"
+
+hypershift install --hypershift-image $HYPERSHIFT_IMAGE
+oc -n hypershift get pods -w
+```
+
+### Deploy Hyperhisft cluster
+```bash
+# Create namespace
+oc create namespace $AGENT_NAMESPACE
+
+# Create hypershift cluster
+hypershift create cluster agent --name $CLUSTERNAME --base-domain $BASEDOMAIN --pull-secret /tmp/pull-secret.json  --ssh-key $SSHKEY --agent-namespace $AGENT_NAMESPACE --namespace $AGENT_NAMESPACE
+
+# Wait for hypershift cluster (wait for available)
+oc get po -n $CLUSTER_NAMESPACE -w
+oc get hostedclusters -A -w
+
+# Generate kubeconfig for the generated spoke (hypershift) cluster 
+hypershift create kubeconfig --name $CLUSTERNAME --namespace $AGENT_NAMESPACE | sed s/admin/$CLUSTERNAME/g > /tmp/kubeconfig.$CLUSTERNAME
+
+# save the generated kubeconfig in a secret on the hub cluster
+oc -n $CLUSTERNAME create secret generic $CLUSTERNAME-kubeconfig --from-file=kubeconfig=/tmp/kubeconfig.$CLUSTERNAME
+```
+
+## Install CRDS
+```bash
+# install assisted-service CRDs on the spoke cluster
+oc --kubeconfig /tmp/kubeconfig.$CLUSTERNAME create namespace $CLUSTERNAME
+KUBECONFIG=/tmp/kubeconfig.$CLUSTERNAME ENABLE_KUBE_API=true make deploy-resources
+
+# install hive CRDs on the hub cluster
+oc apply -f hive.openshift.io_clusterdeployments.yaml
+oc apply -f hive.openshift.io_clusterimagesets.yaml
+oc apply -f metal3.io_baremetalhosts.yaml
+
+# install hive CRDs on the spoke cluster
+oc --kubeconfig /tmp/kubeconfig.$CLUSTERNAME apply -f hive.openshift.io_clusterdeployments.yaml
+oc --kubeconfig /tmp/kubeconfig.$CLUSTERNAME apply -f hive.openshift.io_clusterimagesets.yaml
+oc --kubeconfig /tmp/kubeconfig.$CLUSTERNAME apply -f metal3.io_baremetalhosts.yaml
+```
+
+## Infrastructure Operator
+### Install the operator
+```bash
+# install OLM (change versions as they progress)
+# download the installation script
+https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.22.0/install.sh -o install.sh
```

Comment:
OLM is available by default on ocp>4.0. but anyway, we can refer to the official doc instead (that uses operator-sdk instead of relying on a specific version): https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/install/install.md#openshift

Reply:
fixed. I left the script just in case but also referred to the official installation guide.